### PR TITLE
Add an ez_maker directpins for rp2040

### PR DESCRIPTION
**Pull request by jepler**
## Description

This allows all exposed pins on the Raspberry Pi Pico to be used
as up to 26 individual keys. Keys use a common ground arrangement.

The firmware is also expected to work on generic RP2040 boards, check
manufacturer pinout diagrams or use trial and error to find out the GP#s
of the pins.

Pinging @skullydazed since they are listed as maintainer for the other ez_maker keyboards.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* (none)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ez_maker/directpins/rp2040/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/*
```
Comment (zvecr) [inline keyboards/ez_maker/directpins/rp2040/config.h:1]: This file should be removed

File: keyboards/ez_maker/directpins/rp2040/info.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
+{
+    "manufacturer": "Raspberry Pi",
+    "keyboard_name": "DirectPins RP2040",
+    "maintainer": "jepler",
+    "bootloader": "rp2040",
+    "debounce": 5,
+    "diode_direction": "COL2ROW",
```
Comment (zvecr) [inline keyboards/ez_maker/directpins/rp2040/info.json:7]:
```suggestion
```

File: keyboards/ez_maker/directpins/rp2040/rp2.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/* Copyright 2021 Zach White
```
Comment (zvecr) [inline keyboards/ez_maker/directpins/rp2040/rp2.c:1]: This file should be removed

File: keyboards/ez_maker/directpins/rp2040/rp2.h [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/* Copyright 2021 Zach White
```
Comment (zvecr) [inline keyboards/ez_maker/directpins/rp2040/rp2.h:1]: This file should be removed

File: keyboards/ez_maker/directpins/rp2040/rules.mk [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# MCU name
+MCU = RP2040
+BOOTLOADER = rp2040
```
Comment (zvecr) [inline keyboards/ez_maker/directpins/rp2040/rules.mk:3]:
```suggestion
# This file intentionally left blank
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by jepler** [comment]
Thank you!

**Pull request merged**